src/quick-lint-js/fe/parse-statement.cpp

```diff
@@ -4607,6 +4607,7 @@ void Parser::parse_and_visit_import(
     }
   };
 
+  Source_Code_Span type_span = this->peek().span();
```

Comment:
I'd be happier if this was optional. As written, it looks like `type_span` will always be meaningful, but it's only *sometimes* meaningful (but usually it's garbage).
```suggestion
  std::optional<Source_Code_Span> type_span = std::nullopt;
```

Reply:
made it optional